.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Nginx image creator
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+
+  create-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    steps:
+    - name: CHECKOUT
+      uses: actions/checkout@v3
+
+    - name: INSTALL Node
+      working-directory: ./node
+      run: npm install
+
+    - name: RUN SuperTest API Testing
+      working-directory: ./node
+      run: npm test
+
+    - name: BUILD Angular
+      working-directory: ./angular
+      run: |
+        npm link @angular/cli
+        ng build
+
+    - name: LOGIN to the Container registry
+      uses: docker/login-action@v3.3.0
+      with:
+        registry: ${{ env.REGISTRY }}
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: EXTRACT metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@v5.5.1
+      with:
+        images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+
+    - name: BUILD AND PUSH Docker image
+      uses: docker/build-push-action@v6.9.0
+      with:
+        context: .
+        push: true
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
```

Comment:
Don't see a step to send email notification on build result

Reply:
yes pls create a custom email handler as part of the workflow